### PR TITLE
#8337 - Appium: required apps and usage

### PR DESCRIPTION
**Pull request by Vegebutcher**
this is an article on necessary components to be installed and used for the Appium addon
Appium commands descriptions:
https://github.com/G1ANT-Robot/G1ANT.Addon.Appium/pull/6

### CODE REVIEW

File: appendices/appium-addon.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+5. Copy the path provided in the *Android SDK Location:* field of SDK Manager window:
+
+   ![SDK Path](C:\Users\Darek\Desktop\git\G1ANT.Manual\-assets\sdk-path.png)
+
+6. Open Windows System Properties: press **Win+Pause** keyboard shortcut, click *Advanced system settings* in the left part of the displayed *System* window and on the resulting *Advanced* tab of *System Properties* window click *Environment Variables…* button.
+
+7. In *System variables* section of *Environment Variables* window click *New…* button. The *New System Variable* dialog box will appear. Enter `ANDROID_HOME` as a variable name and paste the path copied from the SDK Manager into the *Variable value* field. Click OK to close the dialog box. Now *Environment Variables* window should contain this entry:
+
+   ![](C:\Users\Darek\Desktop\git\G1ANT.Manual\-assets\environment-variables.png)
```
Comment (WiktoriaPrusik) [inline appendices/appium-addon.md]: noice

**Review by WiktoriaPrusik — CHANGES_REQUESTED** [review]
💟

File: appendices/appium-addon.md (hunk 1 above)
Comment (Vegebutcher) [reply appendices/appium-addon.md]:
```suggestion
   ![Android Environment Variable](../-assets/environment-variables.png)
```

**Review by WiktoriaPrusik — APPROVED** [review]

**Review by przegwar — APPROVED** [review]
Wow, dobra robota 👍

**Pull request merged**